backend/src/main/resources/templates/emails/password-reset.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```

Comment:
I like that this is extracted in new html file 👍 Maybe we can use thymeleaf in future (do not change it now)